### DISCUSSION TITLE
Curated list of AdonisJS Developers for future AdonisJS talent & job board

### DISCUSSION THREAD

**Discussion by britzdylan** [adonisjs/core#3855] [General]
Hi everyone, I started to create a list of curated AdonisJs Developers, if you would like to be included in this please add yourself and use the link below: 

I am building a AdonisJS job board where people can find job and scout AdonisJS Talent 🚀 

**_https://forms.gle/tyW1cLvbssS6A88r8_**

**Comment by Barbapapazes** [comment]
Hello, @thetutlage is also building a job board for AdonisJS.

Can you provide more informations about the forms you're sharing?

**Reply by britzdylan** [reply]
just form that saving to a spreadsheets, does nothing else. I am more focused on a talent board rather than a job board
